source/draw/DrawState.ooc

```diff
@@ -43,11 +45,25 @@ DrawState: cover {
 		this
 	}
 	// Local region
-	setDestination: func (destination: IntBox2D) -> This {
-		this destination = destination
-		this
+	// Precondition: target exists
```

Comment:
If a crash can occur from something that is not given as argument then a warning is needed.